uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
+1. From the command prompt, start the Windows Application Driver service:
+```cmd
+"C:\Program Files\Windows Application Driver\WinAppDriver.exe"
+```
+2. From Visual Studio, navigate to the "Test Explorer" tab and locate the `DevHome.UITest` set of tests.
```

Comment:
There has to be a way to do this from the command line. I imagine the command would look something like `vstest-or-something.exe path/to/DevHome.UITests.dll` or `./path/to/DevHome.UITests.exe`. We should figure that out for when this runs in the pipeline if it doesn't already, and because it would probably make it easy to run on a VM. You just copy the test DLL/EXE and run it there (though that's not ideal for debugging)

Reply:
Yes, a `runsettings` file here is great for testing as we can overwrite the test parameters easily from `dotnet.exe` and from the pipeline YAML task. I added examples on how to do that as part of the last section: "Run tests on a different release of Dev Home (Canary, Prod, Dev)"